db/products.js

```diff
@@ -0,0 +1,48 @@
+const {client} = require('./client');
+
+async function createProduct({ name, description, price, imageURL, inStock, category }) {
+    try {
+      const { rows: [product] } = await client.query(`
+            INSERT INTO products(name, description, price, "imageURL", "inStock", category)
```

Comment:
Have we tested what happens when we call this without all of these fields?  What if `imageURL = ''`...?